src/applications/vaos/tests/project-cheetah/components/VAFacilityPage/index.unit.spec.jsx

```diff
@@ -623,7 +629,13 @@ describe('VAOS vaccine flow: <VAFacilityPage>', () => {
       store,
     });
 
-    await screen.findByText(/We found one VA location for you/i);
+    // it should verify alert heading
+    expect(
+      await screen.findByRole('heading', {
+        level: 2,
+        name: 'We found one VA location for you',
+      }),
+    );
```

Comment:
Generally, we should put `.to.be.ok;` or `to.exist;` at the end of these `expect()` statements. The `expect` statements aren't technically necessary here at all (if `findByRole` fails it'll thrown an error), but if we're going to use them we should fully write out what we're asserting.

Reply:
Makes total sense. Adding the assertion.